base/changes.txt

```diff
@@ -6,6 +6,10 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2024-11-11  Joseph Wright  <Joseph.Wright@latex-project.org>
+	* ltfinal.dtx (subsection{Case changing}):
+	Use \text_titlecase_first:nn for titlecasing
```

Comment:
I tought, that there was some plan to support both "first" and "all" via the optional argument?

Reply:
Plan was to make a change, see if it looks OK, then worry about an option :) I'm not sure what key name to use, honestly.

Reply:
words = first | all  was proposed wasn't it? I think that would be fine

Reply:
Added the new keyword and docs